auth_totp/controllers/main.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime, timedelta
```

Comment:
Ok so import alpha is weird. Take a look at https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports and let me know in chat with questions